docs/source/getting_started_with_liminal.md

```diff
@@ -0,0 +1,153 @@
+
+
+# Getting Started with Apache Liminal
+
+Liminal is a MachineLearning orchestration platform, or in other words: Liminal enables data scientists and data engineers to define their entire ML system and flow using configuration. From feature engineering to production monitoring - and the framework takes care of the infra behind the scenes and seamlessly integrates with the infrastructure behind the scenes.
+
+
+
+## Quick Start
+
+So, you’ve heard about liminal and decided you want to give it a try.
+
+This guide will allow you to set up your first apache Liminal environment and allow you to create some simple ML pipelines. These will be very similar to the ones you are going to build for real production scenarios. This is actually the magic behind Liminal.
+
+
+
+## Prerequisites
+
+Python 3 (3.6 and up)
+
+[Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+*Note: make sure your kubernetes cluster is running*
```

Comment:
I think there shouldn't be a space between docker desktop and this note.
Suggestion:
```
Make sure kubernetes cluster is running in docker desktop (or custom kubernetes installation on your machine).
```

Reply:
fixed lingo